README.md

```diff
@@ -121,7 +121,9 @@ print(task.result().measurement_counts)
 Specify which quantum computer hardware to use by changing the value of the `device_arn` to the value for quantum computer to use:
 - **IonQ** "arn:aws:braket:::device/qpu/ionq/ionQdevice"
 - **Rigetti** "arn:aws:braket:::device/qpu/rigetti/Aspen-8"
-- **D-Wave** "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6" (See the next section in this document for more information about using D-Wave.)
+- **D-Wave** (See the next section in this document for more information about using D-Wave.)
```

Comment:
To avoid having to change the README every time there's a new ARN, can we instead refer to the AWS documentation?

Reply:
We want to try to avoid having a chain of links if possible. We can switch this out later if needed.